aws-ebs-csi-driver/values.yaml

```diff
@@ -9,6 +9,10 @@ image:
   tag: "v0.4.0"
   pullPolicy: IfNotPresent
 
+CSIProvisionerImage:
```

Comment:
To keep the casing consistent to other fields, could you rename this to `csiProvisionerImage`? Or `provisionerImage`, which I like better, since every image in the driver is about CSI, no need to duplicate the prefix.

Reply:
As you are going add other sidecar image. How about we move this section under a new `sidecar` field? Something like:

```
sidecars:
  provisionerImage:
    repository: quay.io/k8scsi/csi-provisioner
    tag: "v1.3.0"
  attacherImage:
    repository: quay.io/k8scsi/csi-attacher
    tag: "v1.2.0"
```

So that all the sidecar related stuff are grouped under the `sidecar` field.